shpc/main/container/config.py

```diff
@@ -225,6 +225,11 @@ def set_tag(self, tag):
         # this value will be none (and we can raise an error)
         else:
             self.tag = self.tags.get(tag)
+            if not self.tag:
```

Comment:
I think the intention of having none here is that (in other places) we can receive the value as None and then act accordingly (read the previous comment) - the best action may not always be to exit. By hard coding an exit in the function, we don't ever allow that.